client/src/http/index.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+const PORT = process.env.PORT || 7070;
```

Comment:
Using `process.env.PORT` for the client-side code might not work as expected because environment variables are typically set on the server side. Ensure that the `PORT` variable is correctly set and accessible in your client-side environment, or consider using a different method to configure the port for development and production environments.